### PR TITLE
add day2 sriov and ptp default provisioning

### PR DESCRIPTION
**Pull request by jnunyez**
day-2 installation example of sriov operator from a workload cluster like volante in cars.lab domain

### CODE REVIEW

File: components/apps/sriov-operator/overlays/default/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     - op: replace
+       path: '/spec/enableOperatorWebhook'
+       value: true
+  - target:
+      kind: OperatorGroup
+      namespace: openshift-sriov-network-operator
+    patch: |-
+     - op: replace
+       path: '/metadata/name'
+       value: 'sriov-network-operators'
```
Comment (novacain1) [inline components/apps/sriov-operator/overlays/default/kustomization.yaml]: Great to see for disconnected use case.

File: components/apps/ptp-operator/overlays/default/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+     - op: replace
+       path: '/spec/ptpEventConfig/transportHost'
+       value: "amqp://amq-interconnect.openshift-ptp.svc.cluster.local"
+  - target:
+      kind: OperatorGroup
+      namespace: openshift-ptp
+    patch: |-
+     - op: replace
+       path: '/metadata/name'
+       value: 'ptp-operators'
```
Comment (novacain1) [inline components/apps/ptp-operator/overlays/default/kustomization.yaml]: Should this have been in the other commit I reviewed around PTP?

File: clusters/overlays/volante/kustomization.yaml [hunk 3]
```diff
@@ -3,4 +3,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 bases:
-  - ../../../components/apps/sriov-operator
+  - ../../../components/apps/sriov-operator/overlays/default
```
Comment (novacain1) [inline clusters/overlays/volante/kustomization.yaml]: Should we align the naming of the application with the name of the Operator?

SR-IOV = openshift-sriov-network-operator
PTP = openshift-ptp

**Review by novacain1 — COMMENTED** [review]
Left a few comments here, mainly around nomenclature.

File: components/apps/ptp-operator/overlays/default/kustomization.yaml (hunk 2 above)
Comment (jnunyez) [reply components/apps/ptp-operator/overlays/default/kustomization.yaml]: changed the name of the PR.

File: components/apps/sriov-operator/overlays/default/kustomization.yaml (hunk 1 above)
Comment (jnunyez) [reply components/apps/sriov-operator/overlays/default/kustomization.yaml]: Will get there but not in this PR.

File: clusters/overlays/volante/kustomization.yaml (hunk 3 above)
Comment (jnunyez) [reply clusters/overlays/volante/kustomization.yaml]: Agree. Done.

**Comment by jnunyez** [comment]
requires rebase

**Review by novacain1 — CHANGES_REQUESTED** [review]
LGTM, squash commits and I think this will be good.

**Comment by jnunyez** [comment]
> LGTM, squash commits and I think this will be good.

Looks Ok. Ready for merge.

**Review by novacain1 — APPROVED** [review]
looks good

**Pull request merged**